### PR TITLE
BREW-55 / Kobold Press - Deep Magic

### PR DESCRIPTION
**Pull request by Spappz**
For some reason I decided that doing this was a good idea. There were two files in the repo for different datatypes; both quite a bit of a mess tbh (🥴). Luckily, spamming regex to fix typos and other problems is easier than manually converting 700+ spells

- [x] Merge files
- [x] Unconverted stuff hooray
  - [x] Book
    - [x] Cover, introduction, etc.
    - [x] Class stuff
    - [x] Rule expansions
    - [x] Appendices
  - [x] Images
    - [x] Extract 
    - [x] Attach images to spells
  - [x] Wizard companions/familiars
  - [x] Tables
  - [x] Variant rules
  - [x] Hieroglyphs
  - [x] Runes
  - [x] Traps
- [x] Apply latest errata
- [x] Tidy everything up
- [x] Update all references in repo to new file

### CODE REVIEW

File: creature/Kobold Press; Tome of Beasts 3.json [hunk 1]
```diff
@@ -84627,7 +84627,7 @@
 						{
 							"type": "item",
 							"name": "Enhanced Spells.",
-							"entry": "Color-based spells (such as {@spell hypnotic pattern}, {@spell frenzied bolt|DMS} from {@loader Deep Magic|collection/Kobold Press; Deep Magic for 5th Edition.json}, or any {@filter \"prismatic\"|spells|search=prismatic} spell) cast within 1 mile of a prismatic dragon's lair become enhanced. When rolling damage for a spell of that type that deals damage, treat any 1 on a damage die as a 2 and increase the spell save DC by 1 (this doesn't affect the dragon's spells)."
+							"entry": "Color-based spells (such as {@spell hypnotic pattern}, {@spell frenzied bolt|KPDM} from {@loader Deep Magic|collection/Kobold Press; Deep Magic for 5th Edition.json}, or any {@filter \"prismatic\"|spells|search=prismatic} spell) cast within 1 mile of a prismatic dragon's lair become enhanced. When rolling damage for a spell of that type that deals damage, treat any 1 on a damage die as a 2 and increase the spell save DC by 1 (this doesn't affect the dragon's spells)."
```
Comment (revilowaldow) [inline creature/Kobold Press; Tome of Beasts 3.json:84630]:
```suggestion
							"entry": "Color-based spells (such as {@spell hypnotic pattern}, {@spell frenzied bolt|KPDM} from {@loader Deep Magic|collection/Kobold Press; Deep Magic.json}, or any {@filter \"prismatic\"|spells|search=prismatic} spell) cast within 1 mile of a prismatic dragon's lair become enhanced. When rolling damage for a spell of that type that deals damage, treat any 1 on a damage die as a 2 and increase the spell save DC by 1 (this doesn't affect the dragon's spells)."
```

**Comment by Spappz** [comment]
appears it's not liking my `statblock` object? 🤔
will have to investigate tomorrow

**Comment by revilowaldow** [comment]
> appears it's not liking my `statblock` object? 

`tag` not `prop` for `statblock`s for everything except subclasses.
https://github.com/TheGiddyLimit/5etools-utils/blob/ca8f171b15ea3504bb5a8ccf73abaa7dc1ee7eb2/schema/brew-fast/entry.json#L3134

**Comment by Spappz** [comment]
huh, that's weird, especially since it still renders correctly

**Review by TheGiddyLimit — APPROVED** [review]

**Pull request merged**